### PR TITLE
cloudformation: fix changeset_name bug fix #27487

### LINKED ISSUE #27487: Its impossible use changeset_name parameter in cloudformation module
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

cloudformation

##### ANSIBLE VERSION

```
ansible 2.4.0 (devel 97aaf103e8) last updated 2017/07/31 15:29:18 (GMT +300)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/admin/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/admin/ansible/lib/ansible
  executable location = /home/admin/ansible/bin/ansible
  python version = 2.7.5 (default, Sep 15 2016, 22:37:39) [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
```

##### OS / ENVIRONMENT

N/A
##### SUMMARY

Using "changeset_name" parameter results in error:

"msg": "local variable 'changeset_name' referenced before assignment local variable 'changeset_name' referenced before assignment - <type 'exceptions.UnboundLocalError'>"

##### STEPS TO REPRODUCE

```yaml
- name: Creating/Updating stack
  cloudformation:
    stack_name:   "{{stack_name}}"
    profile:      "{{profile}}"
    region:       "{{region}}"
    state:        "{{stack_state}}"
    template_url: "test-change-set.yaml"
    changeset_name: test1
    create_changeset: true
  register:     "{{stack_name}}"
```

##### EXPECTED RESULTS

change set is created with predefined name
##### ACTUAL RESULTS

Fails with error above

```

```

### PR DESCRIPTION
**Pull request by gregory-dan**
##### SUMMARY

bug fix #27487 

##### ISSUE TYPE

 - Bugfix Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by tedder** [comment]
cc @wdegeus can you take a look at this?

**Comment by wdegeus** [comment]
Looks good, ship it.

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
shipit

File: lib/ansible/modules/cloud/amazon/cloudformation.py [hunk 1]
```diff
@@ -333,6 +333,9 @@ def create_changeset(module, stack_params, cfn):
 
             changeset_name = 'Ansible-' + stack_params['StackName'] + '-' + sha1(to_bytes(json_params, errors='surrogate_or_strict')).hexdigest()
             stack_params['ChangeSetName'] = changeset_name
+        else:
+            changeset_name=stack_params['ChangeSetName']
```
Comment (ryansb) [inline lib/ansible/modules/cloud/amazon/cloudformation.py]: Needs to be `changeset_name = stack_params['ChangeSetName']` to match PEP8 style of whitespace around operators.

**Review by ryansb — CHANGES_REQUESTED** [review]

**Review by ryansb — APPROVED** [review]

**Comment by ryansb** [comment]
Thanks @gregory-dan!

**Pull request merged**